Jenkinsfile

```diff
@@ -119,7 +119,6 @@ def docker_build_inside_image( def build_image, String inside_args, String platf
             cd ${build_dir_rel}
             make install -j\$(nproc)
             make build_tests -j\$(nproc)
-            ctest -E hipEnvVarDriver
```

Comment:
I think you just deleted all testing :)  You probably need to replace with 'make test'